backend/fleet_management/permissions.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import Group
+from rest_framework import permissions
+from .constants import Groups
+
+
+def is_in_group(user, group_name):
+    try:
+        return Group.objects.get(name=group_name).user_set.filter(id=user.id).exists()
```

Comment:
That's a potentially slow query which will be called very often due to the idea of groups acting as a permissions.
It's a reverse foreign key relation spanning across two tables. I'd like to at least see it cached somewhere or if that's not possible at the moment in reasonable way it might be good to rewrite it to something easier.

Here are some tips on that (maybe `select_related()`?):
https://medium.com/@ryleysill93/basic-performance-optimization-in-django-ebd19089a33f

Future idea: benchmarks!

Reply:
I am not sure - let's leave it for later :)

Reply:
True, it's good enough considering current load.